Sources/Operators.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 
 // MARK: - <| Operator
 
-infix operator <| { associativity left precedence 150 }
+infix operator <|
```

Comment:
This was a little strange to me. The body is apparently ignored now so the error says to not use a body. How else can we define the associativity and precedence? I didn't find much documentation yet.